x-pack/plugins/fleet/common/constants/preconfiguration.ts

```diff
@@ -28,9 +30,18 @@ export const AUTO_UPGRADE_POLICIES_PACKAGES = autoUpgradePoliciesPackages.map((n
   version: PRECONFIGURATION_LATEST_KEYWORD,
 }));
 
+export const DEFAULT_PACKAGES = [
```

Comment:
Do we want to use a different name here since `DEFAULT_PACKAGES` has some historical context in the project? Maybe `MANAGED_PACKAGES` or `FLEET_PACKAGES`?